pl_bolts/utils/semi_supervised.py

```diff
@@ -51,19 +53,19 @@ def balance_classes(X: np.ndarray, Y: list, batch_size: int):
     nb_batches = math.ceil(len(Y) / batch_size)
 
     # sort by classes
-    final_batches_x = [[] for i in range(nb_batches)]
-    final_batches_y = [[] for i in range(nb_batches)]
+    final_batches_x: List[Any] = [[] for i in range(nb_batches)]
+    final_batches_y: List[Any] = [[] for i in range(nb_batches)]
 
     # Y needs to be np arr
     Y = np.asarray(Y)
 
     # pick chunk size for each class using the largest split
-    chunk_size = []
+    chunk_size_class = []
```

Comment:
@akihironitta, `mypy` was giving error if I don't change `chunk_size` to something else because it is `List[int]` and when we do `chunk_size = max(chunk_size)` it is becoming `int`. Let me know if this okay to do.

Reply:
I think you're right. How about renaming the variable `chunk_size_class` to `chunk_sizes`?